src/TreeView/TreeView.tsx

```diff
@@ -362,6 +362,23 @@ const LinkItem: React.FC<TreeViewLinkItemProps> = ({href, onSelect, ...props}) =
   )
 }
 
+// ----------------------------------------------------------------------------
+// TreeView.LoadingItem
+
+const LoadingItem: React.FC = () => {
+  return (
+    // TODO: What aria attributes do we need to add here?
+    <Item>
```

Comment:
🙋 What aria props should we add to the LoadingItem? cc @ericwbailey

Reply:
I'm in the process of testing this out, but I believe we'll need a combination of `aria-hidden` (to hide decorative nodes), `aria-busy` (to communicate loading state), and `aria-live` (to announce state change).